pkg/apis/edgefs.rook.io/v1alpha1/placement.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v1alpha1
+
+import (
+	rook "github.com/rook/rook/pkg/apis/rook.io/v1alpha2"
+)
+
+const (
+	PlacementKeyMgr    = "mgr"
+	PlacementKeyTarget = "target"
+)
+
+// GetMgrPlacement returns the placement for the MGR service
+func GetMgrPlacement(p rook.PlacementSpec) rook.Placement {
+	return p.All().Merge(p[PlacementKeyMgr])
+}
+
+// GetOSDPlacement returns the placement for the Target service
```

Comment:
GetOSD* looks Ceph related, do both of these placement functions make sense for EdgeFS?